rtf_vis_tool/src/App.js

```diff
@@ -3,13 +3,13 @@ import {extend} from "react-three-fiber";
 import {OrbitControls} from 'three/examples/jsm/controls/OrbitControls';
 import './App.css';
 
-import DivGraph from './Components/DivGraph';
+import LandingPageGraph from './Components/LandingPageGraph';
```

Comment:
can we add a unit test?

https://testing-library.com/docs/react-testing-library/intro/
https://itnext.io/how-to-unit-test-in-react-72e911e2b8d

Reply:
I've added some unit tests for smaller, helper components that do not involve React Three Fiber. This is because the React unit testing framework, Enzyme, does not provide native support for third party packages like React Three Fiber. Thus, it's incompatible with components with render 3D-related components.